custom-crawls/README.md

```diff
@@ -0,0 +1,42 @@
+# Tutorial: Custom Crawlers
+
+_Note: This tutorial is a work in progress. Please add your feedback to [datatogether/learning](https://github.com/datatogether/learning/issues)!_
+
+## Prerequisites
+
+* You care about a dataset that exists on the web and downloading the data could be benifit from some automation of downloading
+
+## Learning Objectives
+
+After going through this tutorial you will know
+
+* What a custom crawler is and why some websites need one
+* What should your custom crawler extract from a webpage?
```

Comment:
Can we have this second point phrased not as a question, so 
something like: "What your custom crawler needs to extract from a web page"